package.json

```diff
@@ -79,11 +87,13 @@
     "@octokit/rest": "^21.0.2",
     "@octokit/types": "^13.6.1",
     "@octokit/webhooks": "^13.3.0",
-    "@sinclair/typebox": "^0.33.21",
     "@ubiquity-os/ubiquity-os-logger": "^1.3.2",
     "dotenv": "^16.4.5",
     "hono": "^4.6.9"
   },
+  "peerDependencies": {
+    "@sinclair/typebox": "0.34.3"
```

Comment:
peerDeps are great but yarn does not resolve them automatically afaik. Why did you move it here?

Reply:
As I understand it will warn the user if they are trying to install a typebox with version that doesn't match SDK's version

Reply:
Correct but then it is the responsibility of the user to install TypeBox. However since we use it everywhere I believe it should be fine.